### PR TITLE
Design review of contextual help

### PR DESCRIPTION
**Pull request by jyothishnt**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1320

## Description
This PR contains design review tweaks.
I have moved HistoryButtons into a separate component from HelpPopupBody.

## Deployment URL
http://contextual-help-design-review.review.ensembl.org/

## Views affected
Help article popup

### Other effects
NA

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications
NA

### CODE REVIEW

File: src/ensembl/src/shared/components/help-popup/HelpPopupBody.scss [hunk 1]
```diff
@@ -60,3 +53,13 @@ $heading-outdent: 16px;
...
+
+.aside {
+  grid-column: aside;
+
+  h2 {
+    font-size: 13px;
+    font-weight: $light;
+    margin: 1.5rem 0;
+  }
+}
```
Comment (azangru) [inline src/ensembl/src/shared/components/help-popup/HelpPopupBody.scss:65]: Huh? These styles are reused in the Help app, which is why they were in HelpArticle.scss. Moving them out breaks styles in Help app.

File: src/ensembl/src/shared/components/help-article/HistoryButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { MutableRefObject } from 'react';
+import classNames from 'classnames';
+import HelpPopupHistory from '../help-popup/helpPopupHistory';
```
Comment (azangru) [inline src/ensembl/src/shared/components/help-article/HistoryButtons.tsx:19]: The history, conceptually, belongs to the same folder as the history buttons. Especially if the history is going to be reused in the Help app.

Comment (jyothishnt) [reply src/ensembl/src/shared/components/help-article/HistoryButtons.tsx:19]: from code review: bring back HistoryButton component into help-popup directory. We will take care of reorganising the files in a different ticket as it needs more discussion/thinking.

**Comment by msmanoj** [comment]
@jyothishnt - I think you missed the following from the ticket:
"Remove the 17px from line-height on .textArticle p (line height should revert to 1.5)"
https://github.com/Ensembl/ensembl-client/pull/588/files#diff-4779529293e7b159150b0395a046c4752bbee34a44750d33458dfbb3c953bd03L34

**Review by azangru — APPROVED** [review]

**Pull request merged**